src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <Display />
+      <ButtonPanel />
+    </>
+  );
+}
```

Comment:
## Use ES6 arrow function
- [ ] Kindly change this `App` component to ES6 arrow functional component.
```suggestion
const App = () => {
  return (
    <>
      <Display />
      <ButtonPanel />
    </>
  );
}
```